templates/zerver/invalid_realm.html

```diff
@@ -0,0 +1,9 @@
+{% extends "zerver/portico.html" %}
+{% block portico_content %}
+
+<h3>{{ _('Realm Does Not Exist') }}</h3>
+
+<p>{{ _('Hi there! Thank you for your interest in Zulip') }}.</p>
+<p>{% trans %}The organization you are trying to access does not exist{% endtrans %}.</p>
```

Comment:
I think we should be more bounded in what we're claiming -- how about:

`The organization you are trying to access does not have a Zulip setup at this address.`